### ISSUE TITLE
Smaller map size for a better console visibility

### ISSUE THREAD

**Issue by FabriceFABS** [springweblobby/swl-website#67]
I agree a big map display is nice.
What about making smaller (half size is real enough IMO) map size in the current battle ?
Sentences in the console are not really displayed properly : There's, for most of them, a carriage return sometimes, it seems you see a flooded sentences you don't have time to read.

Have a look the difference between SL & SWL :
<img src="http://img4.hostingpics.net/pics/82905087SL.png">
<img src="http://img4.hostingpics.net/pics/713019SWL.png">

**Comment by AppSynergy** [comment]
I agree. Perhaps simply moving a few things around would give a more balanced look?

![swl-battle](https://cloud.githubusercontent.com/assets/3657244/18143643/e9cca78a-6fba-11e6-813b-f065b2487b3b.png)

for example.

I've been looking at upgrading some more of the css to flex layouts, and that approach seems relevant here too.

I'd also like to find more room for the player lists, maybe two teams per row. Maybe just an issue for games like Zero-K where big team play is a major sport.
